programmers/5일차/이상한-문자-만들기.js

```diff
@@ -0,0 +1,19 @@
+function solution(s) {
+    const splitS = s.split(" ");
+    let changeWord = ""
+
+    for (let word of splitS) {
+        for (let index = 0; index<word.length; index++) {
+            if (index % 2 == 0) {
+                changeWord += word[index].toUpperCase();
+            }
+            else changeWord += word[index].toLowerCase();
+        }
+        changeWord += " ";
+    }
+    return changeWord.slice(0,changeWord.length-1)
+    
+}
```

Comment:
```suggestion

function solution(s) {
    return s.split(' ')
        .map(word => word.split('')
        .map((caseStr, index) => index % 2 ? caseStr.toLowerCase() : caseStr.toUpperCase())
        .join('')).join(' ');
}
```
이건 제 코드는 아니고 옆에 멘토 코드인데  map함수를 중첩시키면 이렇게도 풀 수 있네요  ㅋㅋㅋㅋ  참고해보시면 좋을 것 같아요